### DISCUSSION TITLE
posts/bilibili-cpp-primer-0-1-preface/

### DISCUSSION THREAD

**Discussion by giscus** [sonnycalcr/sonnycalcr.github.io#3] [Announcements]
# posts/bilibili-cpp-primer-0-1-preface/

Hey 朋友们，今天我们来聊一期系列介绍的视频。 谈一谈我接下来将要在 B 站投稿的 C++ Primer 第五版的系列视频。 名字叫：【大学生】《C++ Primer》精读精

https://sonnycalcr.github.io/posts/bilibili-cpp-primer-0-1-preface/

**Comment by wpf-forever** [comment]
催更一下

**Reply by sonnycalcr** [reply]
今天就更，昨天有事耽搁了(手动捂脸
